src/devices/DCMotor/DCMotor.cs

```diff
@@ -116,20 +116,24 @@ public static DCMotor Create(int speedControlPin, GpioController controller = nu
         }
 
         /// <summary>
-        /// Creates <see cref="DCMotor"/> instance which allows to control speed in both directions.
+        /// Creates <see cref="DCMotor"/> instance using two pins which allows to control speed in both directions.
         /// </summary>
         /// <param name="speedControlChannel"><see cref="PwmChannel"/> used to control the speed of the motor</param>
         /// <param name="directionPin">Pin used to control the direction of the motor</param>
+        /// <param name="singleBiDirectionPin">True if a controller with one direction input is used,
+        /// false if a controller with two direction inputs is used</param>
         /// <param name="controller"><see cref="GpioController"/> related to the <paramref name="directionPin"/></param>
         /// <param name="shouldDispose">True to dispose the Gpio Controller</param>
         /// <returns><see cref="DCMotor"/> instance</returns>
         /// <remarks>
-        /// <paramref name="speedControlChannel"/> can be connected to either enable pin of the H-bridge.
-        /// or directly to the input related with the motor (if H-bridge allows inputs to change frequently).
+        /// <paramref name="speedControlChannel"/> should be connected to the one of two inputs
+        /// related with the motor direction (if H-bridge allows inputs to change frequently),
+        /// or to PWM input if a controller with one direction input is used.
         /// <paramref name="directionPin"/> should be connected to H-bridge input corresponding to one of the motor inputs.
+        /// or to DIR input if a controller with one direction input is used.
         /// Connecting motor directly to GPIO pin is not recommended and may damage your board.
         /// </remarks>
-        public static DCMotor Create(PwmChannel speedControlChannel, int directionPin, GpioController controller = null, bool shouldDispose = true)
+        public static DCMotor Create(PwmChannel speedControlChannel, int directionPin, bool singleBiDirectionPin, GpioController controller = null, bool shouldDispose = true)
```

Comment:
new argument should be false by default and be last on the list to allow source compatibility (while we allow for some breaking changes we should not make them unless not necessary and it's not here)

Reply:
I wanted to make future users think twice about what driver they are using 😄, but you are right, compatibility is more important